VMT/templates/KCSA.md

```diff
@@ -0,0 +1,71 @@
+# Kata Containers security advisories (KCSA) template
+
+> **FIXME** This template most likely needs review and rewrite.
+
+The document is first submitted as a markdown formatted description to the KCSA project using this template:
+
+```
+date: YYYY-MM-DD
+
+id: KCSA-$NUM
+
+title: '$TITLE'
+
+description: '$DESCRIPTION_CONTENT'
+
+affected-products:
+
+  - product: $PROJECT
+    version: $AFFECTED_VERSIONS
+
+vulnerabilities:
+
+  - cve-id: $CVE
+
+reporters:
+
+  - name: '$CREDIT'
+    affiliation: $CREDIT_AFFILIATION
+    reported:
+      - $CVE
+
+issues:
+
+  links:
+    - https://github.com/kata-containers/$PROJECT/issues/$ISSUE_NR
+
+ reviews:
+
+  v1.2.3:
+    - https://github.com/kata-containers/$PROJECT/pull/$PR_NUMBER
+
+  v2.3.4:
+    - https://github.com/kata-containers/$PROJECT/pull/$PR_NUMBER
+
+  type: GitHub
+
+notes:
+  - 'Optional note such as cross project version requirements'
+```
+
+Once approved, the request will be added to the KCSA repository and linked from the KCSA inded page. It will then be emailed out.
+We send two separate emails, to avoid off-topic replies to oss-security list:
+
+```
+-   *To:* <openstack-announce@lists.openstack.org>, <openstack@lists.openstack.org>
```

Comment:
We can either direct these to the kata-dev list (indefinitely or temporarily), or set up a new, separate list for these kinds of announcements

Reply:
Ah, my bad - looks like I missed some s/OSF/Kata/ places in the docs. Will fix those up (and in this instance, go with the kata-dev mailing list).

Reply:
this needs updating still.